weblab/templates/includes/navbar.html

```diff
@@ -6,6 +6,7 @@
       <li><a href="{% url 'experiments:list' 'mine' %}">Experiments</a></li>
       <li><a href="{% url 'entities:list' 'model' %}">My models</a></li>
       <li><a href="{% url 'entities:list' 'protocol' %}">My protocols</a></li>
+      <li><a href="{% url 'datasets:list' 'mine' %}">My datasets</a></li>
```

Comment:
I don't think the `'mine'` is wanted here?

Reply:
Ah, no, I see, you're thinking to allow listing not just their own datasets in due course. OK, makes sense.